<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>580</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>770</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableView" name="tableView">
         <property name="sizeIncrement">
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>580</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1500</width>
        <height>1500</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="2">
        <widget class="QPushButton" name="Add">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>Add new song</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="8" column="4">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Maximum Duration</string>
            </property>
           </widget>
          </item>
          <item row="6" column="4">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Maximum Danceability</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="3">
           <widget class="QSpinBox" name="minDur"/>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Artist</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0" colspan="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Minimum Year</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QLineEdit" name="Artist"/>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Minimum Danceability</string>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Maximum Energy</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="3">
           <widget class="QSpinBox" name="minDance"/>
          </item>
          <item row="5" column="3" colspan="2">
           <widget class="QSpinBox" name="maxDance"/>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Minimum Energy</string>
            </property>
           </widget>
          </item>
          <item row="7" column="3" colspan="2">
           <widget class="QSpinBox" name="maxEnergy"/>
          </item>
          <item row="11" column="3" colspan="2">
           <widget class="QSpinBox" name="maxYear"/>
          </item>
          <item row="10" column="4">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Maximum Year</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Minimum Duration</string>
            </property>
           </widget>
          </item>
          <item row="9" column="3" colspan="2">
           <widget class="QSpinBox" name="maxDur"/>
          </item>
          <item row="7" column="0" colspan="3">
           <widget class="QSpinBox" name="minEnergy"/>
          </item>
          <item row="11" column="0" colspan="3">
           <widget class="QSpinBox" name="minYear"/>
          </item>
          <item row="2" column="4">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Genre</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Filters</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3" colspan="2">
           <widget class="QComboBox" name="Genre">
            <item>
             <property name="text">
              <string>All</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="DeleteFromPL">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Delete from Table</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="Playlist">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Playlist</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="SaveCSV">
         <property name="minimumSize">
          <size>
           <width>245</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#SaveCSV{
background-color: transparent;
border-image: url(save.png);
background: none;
border: none;
background-repeat: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="AddToPlaylist">
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#AddToPlaylist {
background-color: transparent;
border-image: url(empty-heart.png);
background: none;
border: none;
background-repeat: none;
}
#AddToPlaylist:pressed
{
   border-image: url(heart.jpg);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="pushButton">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>Edit song</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1056</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
